extensions/interactions/ItemSelectionInput/directives/item-selection-input-validation.service.ts

```diff
@@ -111,7 +110,8 @@ export class ItemSelectionInputValidationService {
   }
 
   getAllWarnings(
-      stateName: string, customizationArgs: any, answerGroups: AnswerGroup[],
+      stateName: string, customizationArgs:
```

Comment:
Please keep parameter names and types on same line.

Reply:
done